spec.bs

```diff
@@ -456,20 +457,22 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     [=auction config/component auctions=].
   1. [=list/For each=] |component| in |auctionConfig|'s [=auction config/component auctions=],
     [=parallel queue/enqueue steps|enqueue the following steps=] to |queue|:
-    1. Let |compWinner| be the result of [=generate and score bids=] with |component|, |global|, and
-      false.
-    1. If |compWinner| is not failure or null, [=score and rank a bid=] with |compWinner|,
-      |leadingBidInfo|, |desisionLogicScript|, and true.
+    1. Let |compWinner| be the result of [=generate and score bids=] with |component|,

```

Comment:
(had a question about the type of `|component|, but I see that each item in `[=auction config/component auctions=]` is an `auctionConfig`.)

Reply:
yes, correct.